### ISSUE TITLE
only works with libfmt 10

### ISSUE THREAD

**Issue by Dlay2017** [cemu-project/Cemu#1338]
### Current Behavior

cemu only works with libfmt v10

### Expected Behavior

cemu works with libfmt v11

### Steps to Reproduce

try to install on an actual arch linux

### System Info (Optional)

OS: Arch Linux
GPU:

**Comment by kescherCode** [comment]
Are you experiencing a build failure on [this AUR package](https://aur.archlinux.org/packages/cemu), perhaps?

**Comment by kescherCode** [comment]
For packagers on other distros: Build and install a seperate version 10 package of fmt (example on Arch: [fmt10 AUR package](https://aur.archlinux.org/packages/fmt10)), include that as a dependency for your package, and then simply include this line:

```
sed -i 's/find_package(fmt 9/find_package(fmt 10.0.0...<11.0.0/' CMakeLists.txt
```

in whatever the equivalent of the `prepare()` function in your build script is.

So, @Dlay2017, for you personally, the issue is probably solved for now. In the long-run, yes indeed build compatibility with fmt 11 will need to be figured out, so leave the issue open.

**Comment by Exzap** [comment]
This was fixed in 6dc73f5d797082c25a68ad162377077547948d26
